### ISSUE TITLE
Cannot start nuxt: Cannot read properties of undefined (reading 'push')

### ISSUE THREAD

**Issue by kemalcany** [windicss/nuxt-windicss#186]
A [similar problem](https://github.com/nuxt-modules/tailwindcss/issues/480) as in nuxt-tailwind package occurs with this package as well as of nuxt version 3.0.0-rc.13

```
 ERROR  Cannot start nuxt:  Cannot read properties of undefined (reading 'push')                                                                                                                       20:52:17

  at Object.options.onConfigResolved (node_modules/.pnpm/nuxt-windicss@2.5.2_vite@3.2.2/node_modules/nuxt-windicss/dist/module.mjs:125:32)
  at resolveOptions (node_modules/.pnpm/@windicss+plugin-utils@1.8.8/node_modules/@windicss/plugin-utils/dist/index.mjs:438:88)
  at _init (node_modules/.pnpm/@windicss+plugin-utils@1.8.8/node_modules/@windicss/plugin-utils/dist/index.mjs:937:21)
  at Object.init (node_modules/.pnpm/@windicss+plugin-utils@1.8.8/node_modules/@windicss/plugin-utils/dist/index.mjs:947:21)
  at setup (node_modules/.pnpm/nuxt-windicss@2.5.2_vite@3.2.2/node_modules/nuxt-windicss/dist/module.mjs:158:30)
  at Object.normalizedModule (node_modules/.pnpm/@nuxt+kit@3.0.0-rc.9/node_modules/@nuxt/kit/dist/index.mjs:630:29)
  at async installModule (node_modules/.pnpm/@nuxt+kit@3.0.0-rc.13_rollup@2.79.1/node_modules/@nuxt/kit/dist/index.mjs:427:3)
  at async initNuxt (node_modules/.pnpm/nuxt@3.0.0-rc.13_gypgyaqhine6mwjfvh7icfhviq/node_modules/nuxt/dist/index.mjs:1787:7)
  at async load (node_modules/.pnpm/nuxi@3.0.0-rc.13/node_modules/nuxi/dist/chunks/dev.mjs:6778:9)
  at async Object.invoke (node_modules/.pnpm/nuxi@3.0.0-rc.13/node_modules/nuxi/dist/chunks/dev.mjs:6839:5)
  at async _main (node_modules/.pnpm/nuxi@3.0.0-rc.13/node_modules/nuxi/dist/cli.mjs:50:20)
```

**Comment by barisbora** [comment]
Looks like there is issue on RC13, Downgraded RC13 to RC12 and fixed for now

**Comment by ramsesgarate** [comment]
The same thing happens to me updating to version 13, as @barisbora mentioned I had to go back to version 12

**Comment by harlan-zw** [comment]
Hey guys, thank for reporting this issue. Looks like a bug with Nuxt dropping the nuxt2 config schema in the latest RC.

I've pushed up a fix in 2.5.3, let me know if you have any issues with it
